### DISCUSSION TITLE
dnssec support

### DISCUSSION THREAD

**Discussion by runningman84** [0xERR0R/blocky#156] [Q&A] [answered]
it would be great of blocky would validate dnssec signatures...

**Comment by 0xERR0R** [comment, marked as answer]
blocky uses https://github.com/miekg/dns which supports DNSSec. If I run the test on [DNSSEC Resolver Test](https://dnssec.vs.uni-due.de/) with blocky as resolver, it is successful. But may be this is not a valid test.

**Comment by runningman84** [comment]
In this case you should add that to your feature list.
